composer.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "automattic/crowdsignal-plugin",
+  "description": "Create and manage Crowdsignal polls and ratings in WordPress",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0-or-later",
+  "homepage": "https://crowdsignal.com/",
+  "authors": [
+    {
+      "name": "Automattic, Inc.",
+      "homepage": "https://automattic.com/"
+    }
+  ],
+  "support": {
+    "issues": "https://github.com/automattic/polldaddy-plugin/issues",
+    "source": "https://github.com/automattic/polldaddy-plugin"
```

Comment:
These are incorrect. This repo lives at https://github.com/Automattic/crowdsignal-plugin